package.json

```diff
@@ -12,6 +12,7 @@
     "@babel/core": "^7.8.7",
     "@babel/node": "^7.8.7",
     "@babel/preset-env": "^7.8.7",
+    "@babel/register": "^7.7.4",
```

Comment:
why is this used?

Reply:
@lovesh ?

Reply:
For debugging with IDE. Not needed anymore, will remove.